### PR TITLE
Fix export of grouping modifier

### PR DESCRIPTION
**Pull request by tcolgate**

### CODE REVIEW

**Comment by tcolgate** [comment]
Fixes for #2163

**Comment by beorn7** [comment]
@fabxc @brian-brazil Your opinion is needed.

**Comment by grobie** [comment]
I don't think this change alone is enough to handle the issue. See my
comment in #2163.

**Comment by tcolgate** [comment]
@grobie @brian-brazil so is this change sufficient?

File: promql/printer_test.go [hunk 1]
```diff
@@ -94,7 +94,10 @@ func TestExprString(t *testing.T) {
 			in: `a - ON(b) GROUP_LEFT(x, y) c`,
 		},
 		{
-			in: `a - ON(b) GROUP_LEFT c`,
```
Comment (grobie) [inline promql/printer_test.go]: I wouldn't change the input, but make the new behavior clear by adding the `out` attribute.

Comment (tcolgate) [inline promql/printer_test.go]: Updated, (the indentation changes with the addition out out:)

Comment (tcolgate) [inline promql/printer_test.go]: Updated

**Pull request merged**